### PR TITLE
update changelog for 0.12.2-0ubuntu1 and 0.12.2-0ubuntu2

### PR DESCRIPTION
**Pull request by lyw07**
~~Changes for the 0.12.2~b3 and 0.12.2 releases.~~

Please let me know if you have any questions. Thank you!

EDIT:
Removed the entry 0.12.0~beta3 to move the change entries into 0.12.2-0ubuntu1, as suggested by Benjamin.
Also since I published the 0.12.2-0ubuntu2 to PPA as requested by Devon, I had to change the name from @jredrejo to my name to sign the debian installer.

### CODE REVIEW

**Comment by benjaoming** [comment]
This is fine, but IMO, you can also scratch the beta releases and digest all the change log entries into the stable release. I.e. everything that goes into `kolibri` is a digest of what was pre-released in `kolibri-proposed`.

**Comment by lyw07** [comment]
thank you @benjaoming ! 
So if I'm following it correctly, would the changlog be like this? 
```
kolibri-source (0.12.2-0ubuntu1) trusty; urgency=medium

  * debian/patches/sentry_removal.diff: remove unneeded files from sentry sdk
  * New upstream release

 -- Lingyi Wang <lingyi@learningequality.org>  Mon, 01 Apr 2019 13:04:32 -0400

```

Does this also apply to changelog for future releases? For example, if we have a pre-release 0.12.3~ b2 and a stable release 0.12.3, when we release 0.12.3, we can remove the changelog entry for 0.12.3~b2 and put the changes into the changelog entry stable release 0.12.3.

File: debian/changelog [hunk 1]
```diff
@@ -1,15 +1,15 @@
-kolibri-source (0.12.2-0ubuntu2) trustry; urgency=medium
+kolibri-source (0.12.2-0ubuntu2) trusty; urgency=medium
```
Comment (jredrejo) [inline debian/changelog]: omg!

Comment (lyw07) [reply debian/changelog]: sorry I didn't review your PR carefully and missed this typo.

Comment (indirectlylit) [reply debian/changelog]: hahha

**Pull request merged**